File: WEBAPI/Models/SluzbenoLice.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json.Serialization;

namespace WEBAPI.Models

{
    [Table("SluzbenoLice")]
    public class SluzbenoLice{

        [Key]
        [Column("ID")]
        public int Id{get;set;}
        [Column("Ime")]
        public string Ime{get;set;}
        [Column("Mesto")]
        public string Mesto{get;set;}
        [Column("Uloga")]
        public string Uloga{get;set;}


        [JsonIgnore]
        public Utakmica Utakmica{get;set;}
        
    }
}

File: FRONT/sluzbenoLice.js
export class SluzbenoLice{

    constructor(ID,ime,mesto,uloga){
        this.ID = ID;
        this.ime = ime;
        this.mesto = mesto;
        this.uloga = uloga;
    }
    getIme(){
        return this.ime;
    }
    getUloga()
    {
        return this.uloga;
    }
    getMesto()
    {
        return this.mesto;
    }
    setID(id)
    {
        this.ID = id;
    }
}